src/CodeGeneration.Roslyn.Tool/CommandLine/Enumerable.cs

```diff
@@ -64,6 +64,11 @@ public static T[] ToArray<T>(this IEnumerable<T> source)
             return System.Linq.Enumerable.ToArray(source);
         }
 
+        public static Dictionary<TKey, TElement> ToDictionary<T, TKey, TElement>(this IEnumerable<T> source, Func<T, TKey> keySelector, Func<T, TElement> elementSelector)
```

Comment:
It's not clear to me why is this needed. Looks like it just duplicates/wraps the same signature as Linq.

Reply:
This was needed to get [this sanitization method](https://github.com/AArnott/CodeGeneration.Roslyn/pull/210/files/9bfa30c5a7f343b7be84976a72399ff8e8de69be#diff-9070cd2b914605c42c39f67b4431b352R140-R143) to work.  I tried adding a using for System.Linq in that Program.cs file, but got a name clash between linq enumerables and codegeneration.roslyn.tool.commandline enumerables.  I could use the fully qualified linq name in the Program.cs file, but this seemed cleaner and more consistent.

Reply:
Yeah we should probably get rid of that commandline stuff somehow :/